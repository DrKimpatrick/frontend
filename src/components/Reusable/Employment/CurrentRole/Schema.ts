import { object, string } from 'yup';

export const currentRoleSchema = object().shape({
  companyName: string().required('company name is required'),
  supervisor: string().required('supervisor is required'),
  title: string().required('title is required'),
  startDate: string().required('start date is required')
});
