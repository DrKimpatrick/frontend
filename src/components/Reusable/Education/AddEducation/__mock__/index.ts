export const initialState = {
  users: {
    currentUser: {
      isLoggedIn: true,
      loading: false,
      error: null,
      message: null,
      created: false,
      data: {
        firstName: 'John',
        lastName: 'Smith',
        profilePicture:
          'https://cdn.pixabay.com/photo/2015/10/05/22/37/blank-profile-picture-973460_960_720.png'
      }
    },
    user: {
      _id: 'id',
      username: 'username',
      email: 'email',
      roles: ['talent'],
      updatedAt: '2020-10-10',
      dateRegistered: '2020-10-10',
      paymentStatus: 'verified',
      featureChoice: 'local',
      verified: true
    }
  },
  messages: {
    message: 'sent'
  },
  educations: {
    loading: false,
    errors: {
      errors: null
    },
    education: {
      _id: 'id',
      schoolName: 'school name',
      level: 'level',
      degreeOrCertification: 'education',
      specializations: ['specialization'],
      startDate: '2020-10-10',
      endDate: '2020-10-5',
      accomplishments: ['snow'],
      isCurrentEducation: false,
      verificationStatus: 'verified'
    }
  }
};
