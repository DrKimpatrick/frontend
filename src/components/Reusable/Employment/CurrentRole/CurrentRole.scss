@import '../../../Talent/Dashboard/Employment/Employment.scss';

.current-role-section {
  .title {
    font-size: 22px;
    color: #747474;
    text-decoration: none solid rgb(116, 116, 116);
  }

  .back-arrow {
    position: absolute;
    color: #747474;
    left: -43px;
    top: 0px;
    font-size: 22px;
  }
}

@media screen and (max-width: 786px) {
  .current-role-section {
    width: 100%;
    .back-arrow {
      display: none;
    }

    .containers {
      width: 60%;
    }

    .title {
      text-align: center;
      margin-left: 0 !important;
    }
  }
}

@media screen and (max-width: 500px) {
  .current-role-section {
    .containers {
      width: 100%;
    }
  }
}
