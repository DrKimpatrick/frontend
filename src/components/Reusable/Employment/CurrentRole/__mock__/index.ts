export const initialState = {
  users: {
    currentUser: {
      isLoggedIn: true,
      loading: false,
      error: null,
      message: null,
      created: false,
      data: {
        firstName: 'John',
        lastName: 'Smith',
        profilePicture:
          'https://cdn.pixabay.com/photo/2015/10/05/22/37/blank-profile-picture-973460_960_720.png'
      }
    },
    user: {
      _id: 'id',
      username: 'username',
      email: 'email',
      roles: ['talent'],
      updatedAt: '2020-10-10',
      dateRegistered: '2020-10-10',
      paymentStatus: 'verified',
      featureChoice: 'local',
      verified: true
    }
  },
  employments: {
    employment: {
      companyName: 'snow'
    },
    errors: null,
    loading: false
  },
  messages: {
    message: 'sent'
  },
  educations: {
    education: {
      schoolName: 'schoolName'
    }
  }
};
