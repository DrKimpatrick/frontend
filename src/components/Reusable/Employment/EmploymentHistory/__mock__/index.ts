export const initialState = {
  users: {
    currentUser: {
      isLoggedIn: true,
      loading: false,
      error: null,
      message: null,
      created: false,
      data: {
        firstName: 'John',
        lastName: 'Smith',
        profilePicture:
          'https://cdn.pixabay.com/photo/2015/10/05/22/37/blank-profile-picture-973460_960_720.png'
      }
    },
    user: {
      _id: 'id',
      username: 'username',
      email: 'email',
      roles: ['talent'],
      updatedAt: '2020-10-10',
      dateRegistered: '2020-10-10',
      paymentStatus: 'verified',
      featureChoice: 'local',
      verified: true
    }
  },
  employments: {
    employment: {
      companyName: 'snow',
      supervisor: 'supervisor',
      title: 'title',
      startDate: '2020-10-5',
      responsibilities: ['res'],
      accomplishments: ['res'],
      favoriteProject: 'res',
      skillsUsed: []
    },
    errors: null,
    loading: false
  },
  messages: {
    message: 'sent'
  },
  educations: {
    education: {
      schoolName: 'schoolName'
    }
  },
  skills: {
    userSkill: [
      {
        supervisor: {
          detail: {
            name: 'user',
            email: 'email@gmail.com',
            phoneNumber: '0786601003'
          },
          name: 'HR'
        },
        skillsUsed: [],
        responsibilities: [],
        accomplishments: [],
        verificationStatus: 'unverified',
        isCurrentPosition: true,
        _id: '5fcf65a2c4c797a20acc1c99',
        title: 'Product Manager',
        startDate: '2020-10-15T00:00:00.000Z',
        endDate: null,
        companyName: 'heza',
        favoriteProject: 'tembea',
        userId: '5fcdf47ea1d1dd1564a21613',
        createdAt: '2020-12-08T11:38:10.729Z',
        updatedAt: '2020-12-08T11:38:10.729Z'
      }
    ]
  }
};
