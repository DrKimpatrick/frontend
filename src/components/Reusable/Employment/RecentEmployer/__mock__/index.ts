export const initialState = {
  users: {
    currentUser: {
      isLoggedIn: true,
      loading: false,
      error: null,
      message: null,
      created: false,
      data: {
        firstName: 'John',
        lastName: 'Smith',
        profilePicture:
          'https://cdn.pixabay.com/photo/2015/10/05/22/37/blank-profile-picture-973460_960_720.png'
      }
    },
    user: {
      _id: 'id',
      username: 'username',
      email: 'email',
      roles: ['talent'],
      updatedAt: '2020-10-10',
      dateRegistered: '2020-10-10',
      paymentStatus: 'verified',
      featureChoice: 'local',
      verified: true
    }
  },
  employments: {
    employment: {
      companyName: 'snow',
      supervisor: 'supervisor',
      title: 'title',
      startDate: '2020-10-5',
      responsibilities: ['res'],
      accomplishments: ['res'],
      favoriteProject: 'res',
      _id: 'id'
    },
    errors: null,
    loading: false,
    submitLoading: false
  },
  messages: {
    message: 'sent'
  },
  educations: {
    education: {
      schoolName: 'schoolName'
    }
  },
  skills: {
    userSkill: [
      {
        skill: {
          _id: '5fcf1c5b11c16c41f7c6c723',
          skill: 'Python',
          createdAt: '2020-12-08T06:25:31.128Z',
          updatedAt: '2020-12-08T06:25:31.128Z'
        },
        level: 'intermediate',
        verificationStatus: 'verified',
        _id: '5fcf52105cd56b9843252696',
        createdAt: '2020-12-08T10:14:40.231Z',
        updatedAt: '2020-12-08T13:05:46.873Z'
      }
    ]
  }
};
