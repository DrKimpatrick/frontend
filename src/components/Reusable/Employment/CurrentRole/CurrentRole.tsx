import React, { FC, useEffect } from 'react';
import { Link } from 'react-router-dom';
import { useDispatch, useSelector } from 'react-redux';
import { Formik } from 'formik';
import Select from 'react-select';
import { map, get } from 'lodash';
import ArrowBackTwoToneIcon from '@material-ui/icons/ArrowBackTwoTone';
import { ArrowRightAltTwoTone } from '@material-ui/icons';
import { RootState } from 'redux/store';
import { TalentProcess } from 'redux/action-types/user';
import { addEmployment, listEmployments } from 'redux/actions/employment';
import { currentRoleSchema } from './Schema';
import './CurrentRole.scss';

interface Props {
  setPreviousStep: (value: string) => void;
}

const CurrentRole: FC<Props> = props => {
  const { setPreviousStep } = props;

  const dispatch = useDispatch();

  const reducer = useSelector((state: RootState) => {
    const { loading, errors, employment } = state.employments;

    const { message } = state.messages;

    const { user } = state.users;

    return { message, loading, errors, employment, user };
  });

  const getErrors = (field: string) => {
    const { errors } = reducer;

    if (
      errors &&
      errors.errors &&
      Array.isArray(errors.errors) &&
      errors.errors.length > 0 &&
      map(errors.errors, field)
    ) {
      return (
        <>
          {map(map(errors.errors, field), (item, i) => (
            <div key={i} className="inputError">
              {item}
            </div>
          ))}
        </>
      );
    }
    return null;
  };

  useEffect(() => {
    listEmployments()(dispatch);
  }, [dispatch]);

  return (
    <>
      <section className="current-role-section w-1/3 m-auto text-gray-texts">
        <div className="containers">
          <div className="recent-employer-section">
            <div className="flex relative h-auto my-8">
              <div className="back-arrow cursor-pointer">
                <ArrowBackTwoToneIcon />
              </div>
              <h1 className="font-bold text-xl title">
                Tell us how awesome you are!
              </h1>
            </div>
            <Formik
              validateOnChange={false}
              validationSchema={currentRoleSchema}
              initialValues={{
                companyName: '',
                supervisor: {
                  name: '',
                  detail: {
                    name: '',
                    email: '',
                    phoneNumber: ''
                  }
                },
                title: '',
                startDate: '',
                isCurrentPosition: true,
                showDetail: false,
                currentSupervisor: ''
              }}
              onSubmit={values => {
                if (reducer.user) {
                  addEmployment({
                    ...values,
                    user: reducer.user,
                    profileProcess: TalentProcess.SkillRanking
                  })(dispatch);
                }
              }}
            >
              {formik => {
                const { errors, values } = formik;
                return (
                  <form
                    onSubmit={formik.handleSubmit}
                    autoComplete="off"
                    data-testid="submit-form"
                  >
                    <div className="text-gray-texts mt-8">
                      <label>What is your company name ? </label>
                      <input
                        type="text"
                        className="border outline-none bg-transparent rounded-sm w-full px-3 text-gray-texts input-height mt-2"
                        placeholder="Company Name"
                        name="companyName"
                        value={values.companyName}
                        onChange={formik.handleChange}
                      />
                      {errors && errors.companyName && (
                        <div className="inputError">{errors.companyName}</div>
                      )}
                      {!errors ||
                        (!errors.companyName && getErrors('companyName'))}
                    </div>
                    <div className="text-gray-texts mt-4">
                      <label>What is your current role?</label>
                      <input
                        type="text"
                        className="border outline-none bg-transparent rounded w-full px-3 text-gray-texts input-height mt-2"
                        placeholder="Your Title"
                        value={values.title}
                        onChange={formik.handleChange}
                        name="title"
                      />
                      {errors && errors.title && (
                        <div className="inputError">{errors.title}</div>
                      )}
                      {!errors || (!errors.title && getErrors('title'))}
                    </div>
                    <div className="text-gray-texts mt-4">
                      <Select
                        options={[
                          { value: 'Staffing', label: 'Staffing' },
                          { value: 'Employee', label: 'Employee' },
                          { value: 'HR', label: 'HR' }
                        ]}
                        placeholder="Select your supervisor"
                        name="supervisor.name"
                        onChange={v => {
                          formik.setFieldValue(
                            'supervisor.name',
                            (v as any).value,
                            true
                          );
                          formik.setFieldValue('showDetail', true);
                          formik.setFieldValue(
                            'currentSupervisor',
                            (v as any).label
                          );
                        }}
                        values={
                          values.supervisor.name !== ''
                            ? {
                                value: values.supervisor,
                                label: values.supervisor
                              }
                            : null
                        }
                        isMulti={false}
                        styles={{
                          control: base => ({
                            ...base,
                            border: 0,
                            boxShadow: 'none'
                          })
                        }}
                        className="select"
                        defaultValue={
                          values.supervisor.name !== ''
                            ? {
                                value: values.supervisor.name,
                                label: values.supervisor.name
                              }
                            : null
                        }
                      />
                      {errors &&
                        errors.supervisor &&
                        errors.supervisor.name && (
                          <div className="inputError">
                            {errors.supervisor.name}
                          </div>
                        )}
                      {!errors ||
                        (!errors.supervisor && getErrors('supervisor'))}
                      {!errors ||
                        (!errors.supervisor && getErrors('supervisor.name'))}
                    </div>
                    {values.showDetail && (
                      <>
                        <div className="text-gray-texts mt-4">
                          <input
                            type="text"
                            className="border outline-none bg-transparent rounded w-full px-3 text-gray-texts input-height"
                            placeholder={`${values.currentSupervisor} name`}
                            value={get(values.supervisor.detail, 'name', '')}
                            onChange={formik.handleChange}
                            name="supervisor.detail.name"
                          />
                          {errors &&
                            errors.supervisor &&
                            errors.supervisor.detail &&
                            errors.supervisor.detail.name && (
                              <div className="inputError">
                                {errors.supervisor.detail.name}
                              </div>
                            )}
                          {!errors ||
                            (!errors.supervisor &&
                              getErrors('supervisor.detail.name'))}
                        </div>
                        <div className="text-gray-texts mt-4">
                          <input
                            type="text"
                            className="border outline-none bg-transparent rounded w-full px-3 text-gray-texts input-height"
                            placeholder={`${values.currentSupervisor} email`}
                            value={get(values.supervisor.detail, 'email', '')}
                            onChange={formik.handleChange}
                            name="supervisor.detail.email"
                          />
                          {errors &&
                            errors.supervisor &&
                            errors.supervisor.detail &&
                            errors.supervisor.detail.email && (
                              <div className="inputError">
                                {errors.supervisor.detail.email}
                              </div>
                            )}

                          {!errors ||
                            (!errors.supervisor &&
                              getErrors('supervisor.detail.email'))}
                        </div>
                        <div className="text-gray-texts mt-4">
                          <input
                            type="text"
                            className="border outline-none bg-transparent rounded w-full px-3 text-gray-texts input-height"
                            placeholder={`${values.currentSupervisor} phone number`}
                            value={get(
                              values.supervisor.detail,
                              'phoneNumber',
                              ''
                            )}
                            onChange={formik.handleChange}
                            name="supervisor.detail.phoneNumber"
                          />
                          {errors &&
                            errors.supervisor &&
                            errors.supervisor.detail &&
                            errors.supervisor.detail.phoneNumber && (
                              <div className="inputError">
                                {errors.supervisor.detail.phoneNumber}
                              </div>
                            )}
                          {!errors ||
                            (!errors.supervisor &&
                              getErrors('supervisor.detail.phoneNumber'))}
                        </div>
                      </>
                    )}
                    <div
                      className="flex justify-between text-gray-texts mt-4 dateContainer"
                      style={{ flexDirection: 'column', alignItems: 'start' }}
                    >
                      <div className="item w-full h-full pt-2 mt-2">
                        <label>Start Date</label>
                        <input
                          id="date"
                          type="date"
                          name="startDate"
                          onChange={formik.handleChange}
                          value={values.startDate}
                        />
                      </div>
                      {errors && errors.startDate && (
                        <div className="inputError">{errors.startDate}</div>
                      )}
                      {!errors ||
                        (!errors.supervisor && getErrors('startDate'))}
                    </div>
                    <div className="flex justify-center mt-10 flex-col items-center">
                      <button
                        data-testid="next-button"
                        className="next-btn text-white hover:bg-gray-800 font-semibold py-1 px-3 w-32 rounded-sm shadow flex justify-around"
                        type="submit"
                      >
                        <label className="">Next</label>{' '}
                        <ArrowRightAltTwoTone />
                      </button>
                      <Link
                        to="#"
                        className="my-5 font-bold"
                        onClick={() =>
                          setPreviousStep(TalentProcess.SkillRanking)
                        }
                      >
                        Skip
                      </Link>
                    </div>
                  </form>
                );
              }}
            </Formik>
          </div>
        </div>
      </section>
    </>
  );
};

export default CurrentRole;
