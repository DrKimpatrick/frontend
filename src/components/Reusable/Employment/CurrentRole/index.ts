import CurrentRole from './CurrentRole';

export { CurrentRole };
