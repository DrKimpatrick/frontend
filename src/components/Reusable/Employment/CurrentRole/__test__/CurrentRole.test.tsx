import { cleanup, render, fireEvent, waitFor } from '@testing-library/react';
import React from 'react';
import ReactDom from 'react-dom';
import { BrowserRouter as Router } from 'react-router-dom';
import CurrentRole from '../CurrentRole';
import renderer from 'react-test-renderer';
import configureMockStore from 'redux-mock-store';
import thunk from 'redux-thunk';
import { Provider } from 'react-redux';
import { initialState } from '../__mock__';

const mockSetPreviousStep = jest.fn();

let store: any;

const middleware = [thunk];

const mockStore = configureMockStore(middleware);

describe('`CurrentRole` component', () => {
  beforeEach(() => {
    store = mockStore(initialState);
  });

  afterEach(cleanup);

  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDom.render(
      <Provider store={store}>
        <Router>
          <CurrentRole setPreviousStep={mockSetPreviousStep} />
        </Router>
      </Provider>,
      div
    );
  });

  it('matches the snapshot', () => {
    const tree = renderer
      .create(
        <Provider store={store}>
          <Router>
            <CurrentRole setPreviousStep={mockSetPreviousStep} />
          </Router>
        </Provider>
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });

});
