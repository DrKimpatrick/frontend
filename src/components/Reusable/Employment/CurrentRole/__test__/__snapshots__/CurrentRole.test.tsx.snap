// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`\`CurrentRole\` component matches the snapshot 1`] = `
<section
  className="current-role-section w-1/3 m-auto text-gray-texts"
>
  <div
    className="containers"
  >
    <div
      className="recent-employer-section"
    >
      <div
        className="flex relative h-auto my-8"
      >
        <div
          className="back-arrow cursor-pointer"
        >
          <svg
            aria-hidden={true}
            className="MuiSvgIcon-root"
            focusable="false"
            viewBox="0 0 24 24"
          >
            <path
              d="M20 11H7.83l5.59-5.59L12 4l-8 8 8 8 1.41-1.41L7.83 13H20v-2z"
            />
          </svg>
        </div>
        <h1
          className="font-bold text-xl title"
        >
          Tell us how awesome you are!
        </h1>
      </div>
      <form
        autoComplete="off"
        data-testid="submit-form"
        onSubmit={[Function]}
      >
        <div
          className="text-gray-texts mt-8"
        >
          <label>
            What is your company name ? 
          </label>
          <input
            className="border outline-none bg-transparent rounded-sm w-full px-3 text-gray-texts input-height mt-2"
            name="companyName"
            onChange={[Function]}
            placeholder="Company Name"
            type="text"
            value=""
          />
        </div>
        <div
          className="text-gray-texts mt-4"
        >
          <label>
            What is your current role?
          </label>
          <input
            className="border outline-none bg-transparent rounded w-full px-3 text-gray-texts input-height mt-2"
            name="title"
            onChange={[Function]}
            placeholder="Your Title"
            type="text"
            value=""
          />
        </div>
        <div
          className="text-gray-texts mt-4"
        >
          <div
            className="select css-2b097c-container"
            onKeyDown={[Function]}
          >
            <div
              className=" css-c8odyh-control"
              onMouseDown={[Function]}
              onTouchEnd={[Function]}
            >
              <div
                className=" css-g1d714-ValueContainer"
              >
                <div
                  className=" css-1wa3eu0-placeholder"
                >
                  Select your supervisor
                </div>
                <div
                  className="css-b8ldur-Input"
                >
                  <div
                    className=""
                    style={
                      Object {
                        "display": "inline-block",
                      }
                    }
                  >
                    <input
                      aria-autocomplete="list"
                      autoCapitalize="none"
                      autoComplete="off"
                      autoCorrect="off"
                      disabled={false}
                      id="react-select-3-input"
                      onBlur={[Function]}
                      onChange={[Function]}
                      onFocus={[Function]}
                      spellCheck="false"
                      style={
                        Object {
                          "background": 0,
                          "border": 0,
                          "boxSizing": "content-box",
                          "color": "inherit",
                          "fontSize": "inherit",
                          "label": "input",
                          "opacity": 1,
                          "outline": 0,
                          "padding": 0,
                          "width": "1px",
                        }
                      }
                      tabIndex="0"
                      type="text"
                      value=""
                    />
                    <div
                      style={
                        Object {
                          "height": 0,
                          "left": 0,
                          "overflow": "scroll",
                          "position": "absolute",
                          "top": 0,
                          "visibility": "hidden",
                          "whiteSpace": "pre",
                        }
                      }
                    >
                      
                    </div>
                  </div>
                </div>
              </div>
              <div
                className=" css-1hb7zxy-IndicatorsContainer"
              >
                <span
                  className=" css-1okebmr-indicatorSeparator"
                />
                <div
                  aria-hidden="true"
                  className=" css-tlfecz-indicatorContainer"
                  onMouseDown={[Function]}
                  onTouchEnd={[Function]}
                >
                  <svg
                    aria-hidden="true"
                    className="css-6q0nyr-Svg"
                    focusable="false"
                    height={20}
                    viewBox="0 0 20 20"
                    width={20}
                  >
                    <path
                      d="M4.516 7.548c0.436-0.446 1.043-0.481 1.576 0l3.908 3.747 3.908-3.747c0.533-0.481 1.141-0.446 1.574 0 0.436 0.445 0.408 1.197 0 1.615-0.406 0.418-4.695 4.502-4.695 4.502-0.217 0.223-0.502 0.335-0.787 0.335s-0.57-0.112-0.789-0.335c0 0-4.287-4.084-4.695-4.502s-0.436-1.17 0-1.615z"
                    />
                  </svg>
                </div>
              </div>
            </div>
            <input
              name="supervisor.name"
              type="hidden"
              value=""
            />
          </div>
        </div>
        <div
          className="flex justify-between text-gray-texts mt-4 dateContainer"
          style={
            Object {
              "alignItems": "start",
              "flexDirection": "column",
            }
          }
        >
          <div
            className="item w-full h-full pt-2 mt-2"
          >
            <label>
              Start Date
            </label>
            <input
              id="date"
              name="startDate"
              onChange={[Function]}
              type="date"
              value=""
            />
          </div>
        </div>
        <div
          className="flex justify-center mt-10 flex-col items-center"
        >
          <button
            className="next-btn text-white hover:bg-gray-800 font-semibold py-1 px-3 w-32 rounded-sm shadow flex justify-around"
            data-testid="next-button"
            type="submit"
          >
            <label
              className=""
            >
              Next
            </label>
             
            <svg
              aria-hidden={true}
              className="MuiSvgIcon-root"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M16.01 11H4v2h12.01v3L20 12l-3.99-4v3z"
              />
            </svg>
          </button>
          <a
            className="my-5 font-bold"
            href="/"
            onClick={[Function]}
          >
            Skip
          </a>
        </div>
      </form>
    </div>
  </div>
</section>
`;
